tests/FunctionalConnectorTest.php

```diff
@@ -40,6 +40,10 @@ public function connectionToTcpServerShouldSucceedWithLocalhost()
      */
     public function testConnectTwiceWithoutHappyEyeBallsOnlySendsSingleDnsQueryDueToLocalDnsCache()
     {
+        if (DIRECTORY_SEPARATOR === '\\' || defined('HHVM_VERSION')) {
+            $this->markTestSkipped();
```

Comment:
Why is this skipped?

Reply:
if ((DIRECTORY_SEPARATOR === '\\' && PHP_VERSION_ID < 70000) || defined('HHVM_VERSION')) {
            $this->markTestSkipped('Not supported on windows for PHP versions < 7.0 and legacy HHVM');
        }